notebooks/MAST/astroqueryMAST/beginner_program_id_search/beginner_program_id_search.ipynb

```diff
@@ -0,0 +1,402 @@
+    "\n",
+    "And of course, if the need arises, we can logout:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Observations.logout()\n",
+    "session_info = Observations.session_info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Searching for Data by ID"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "We can use a program ID to query the MAST archives for data. In the example below, we'll use 2733 as the ID. This is the program that produced the [stunning images](https://webbtelescope.org/contents/media/images/2022/033/01G709QXZPFH83NZFAFP66WVCZ) of the Southern Ring Nebula!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Let's get a list of all observations associated with this proposal\n",
+    "obs_list = Observations.query_criteria(proposal_id=2733)\n",
+    "\n",
+    "# We can chooose the columns we want to display in our table\n",
+    "disp_col = ['dataproduct_type','calib_level','obs_id','target_name','filters','proposal_pi']\n",
+    "obs_list[disp_col].show_in_notebook(display_length=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have limited the display columns in the above table for conciseness. For a complete list of observation fields (the columns in the above table) and their descriptions, read [here](https://mast.stsci.edu/api/v0/_c_a_o_mfields.html). \n",
+    "\n",
+    "We can verify that we have the right observation by looking at the `'proposal_pi'` column above. The first observation has no PI listed, as it is a release image from the Webb Science Launch; otherwise, it looks like this is the right observation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data Products\n",
+    "\n",
+    "Level 3 products are the result of combining and processing multiple lower level products. These two categories are distinct; level 3 products are target-based (sometimes called source-based), while levels 2 and 1 are directly associated with an exposure. A great starting point to understand JWST files and the processing pipeline is available on the [Jdox website](https://jwst-docs.stsci.edu/understanding-jwst-data-files).\n",
+    "\n",
+    "For level 3 observations, it's likely that there are many associated (levels 2 and 1) data products. Let's take a look at how many products are associated with second observation from our search above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "second_obs = obs_list[1:2]\n",
```

Comment:
If you just want the products assocaited with the 2nd observation in the table, just specify the index directly:

second_obs = obs_list[1]

There is no need to specify a range if extracting one element.  Further, this is dangerous if the order returned by query_criteria ever changes.  If the notebook relies on looking at this specific observation, it would be better to extract the element based on the name of the Observation instead of its relevant position.  If it doesn't matter (e.g., you don't refer to expected number of products, etc.) then it's fine to leave as "the 2nd Observation whatever that ends up being (and hopefully not the OPO one).